src/main.cc

```diff
@@ -108,35 +108,26 @@ int main(int argc, char** argv)
     int ret = ct.checkOptions();
     if (ret < 0) return ret;
 
+    unsigned ngpts[3]    = { ct.ngpts_[0], ct.ngpts_[1], ct.ngpts_[2] };
+    double origin[3]     = { ct.ox_, ct.oy_, ct.oz_ };
+    const double cell[3] = { ct.lx_, ct.ly_, ct.lz_ };
+    Mesh::setup(mmpi.commSpin(), ngpts, origin, cell, ct.lap_type);
+
     mmpi.bcastGlobal(input_filename);
     mmpi.bcastGlobal(lrs_filename);
 
     // Enter main scope
     {
         MGmolInterface* mgmol;
         if (ct.isLocMode())
-            mgmol = new MGmol<LocGridOrbitals>(global_comm, *MPIdata::sout);
+            mgmol = new MGmol<LocGridOrbitals>(global_comm, *MPIdata::sout,
```

Comment:
@jeanlucf22 does this mean we *need* to pass in an lrs file and constraint file to the run command line?

Reply:
we just pass a string. can be an empty string